webserver/website/src/App.vue

```diff
@@ -146,8 +146,8 @@ TrAIn_Connection_Prediction ║   ║
         .then((connections) => {
           this.$store.commit('set_connections', connections)
           this.$store.commit('stop_progress')
-          if (this.$route.path !== '/') {
-            this.$router.push('/')
+          if (this.$route.path !== '/connections') {
```

Comment:
Du hast hiermit irgendwie einen bug eingeführt, dass wenn man noch nichts `connections` in der addressbar hat, dann scrollt man nicht runter

Reply:
Sehr gute beobachtung